mergify/src/main/scala/org/typelevel/sbt/mergify/MergifyAction.scala

```diff
@@ -57,6 +59,21 @@ object MergifyAction {
       }
   }
 
+  final case class RequestReviews(users: List[String] = Nil) extends MergifyAction {
+    override private[mergify] def name = "request_reviews"
+  }
+
+  object RequestReviews {
+    implicit def encoder: Encoder[RequestReviews] =
+      Encoder.forProduct1("users")(_.users)
+  }
+
+  final case class Update() extends MergifyAction
```

Comment:
Would this be better as a `case object`? This is fine, just seems a bit strange :)

Reply:
It might be, yeah. I started off with it as an object and it didn't serialize with the right name, but I figured out how to fix the name on the `RequestReviews`, so it should be doable.